src/components/home/homeComponent/gameCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+class GameCard extends React.Component {
```

Comment:
use functional component instead of class component, if it is not in task (to use class component)